src/api/app/views/webui/obs_factory/staging_projects/_problems.html.erb

```diff
@@ -41,22 +41,18 @@ end -%>
 <% else %>
   <ul class='problem_list'>
     <%- problems.sort! -%>
-    <%- max = 5 -%>
+    <%- num_elements = 6 -%>
     <%- count = problems.size -%>
-    <% if count <= max+1 # Prevent the "1 more" link %>
-      <% problems.each do |p| %>
-        <%= p %>
-      <% end %>
-    <% else %>
-      <% problems[0,max].each do |p| %>
-        <%= p %>
-      <% end %>
+    <% problems[0, num_elements - 1].each do |p| %>
+      <%= p %>
+    <% end %>
+    <% if count > num_elements # Prevent the "1 more" link %>
```

Comment:
I am not sure I understand what you mean :thinking:

Reply:
@bgeuken 

> I mean that when you have to write num_elements - 1 at three places and num_elements at one, you could also reduce the initial value of num_elements to 5 and only have to write num_elements + 1 at one place:-)

but that is exactly what we had before. I find it less readable, because you want to have 6 elements. Then is clearer IMO to do -1 when you are using ranges. Anyway with your suggestion of using `first`, this is solved :bowtie:

Reply:
Fine with me. Was a nitpick anyway:-)